src/main/java/com/gmail/goosius/siegewar/integration/maptowny/MapTownyShortTimeListener.java

```diff
@@ -0,0 +1,19 @@
+package com.gmail.goosius.siegewar.integration.maptowny;
+
+import com.palmergames.bukkit.towny.event.time.NewShortTimeEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class MapTownyShortTimeListener implements Listener {
+
+    private final Runnable renderSieges;
+
+    public MapTownyShortTimeListener(Runnable renderSieges) {
+        this.renderSieges = renderSieges;
+    }
+
+    @EventHandler
+    public void onNewShortTime(NewShortTimeEvent event) {
+        this.renderSieges.run();
```

Comment:
:construction: 
- Have I missed something, or is threading unnecessary here?
- Couldn't we just do something simple like:  SiegeWar.getMapTownyIntegration().displaySieges()  ?